test/components/upgrades.test.js

```diff
@@ -314,6 +327,123 @@ describe('Upgrades testing', function () {
             expect(await this.staking.activeSharesToStake(subjectToActive(0, this.accounts.scanner.address), '10')).to.be.equal('10');
             expect(await this.staking.inactiveSharesToStake(subjectToInactive(0, this.accounts.scanner.address), '10')).to.be.equal('10');
 
+            expect(await this.staking.stakeToActiveShares(subjectToActive(1, AGENT_ID), '10')).to.be.equal('20');
+            expect(await this.staking.stakeToInactiveShares(subjectToInactive(1, AGENT_ID), '10')).to.be.equal('10');
+            expect(await this.staking.activeSharesToStake(subjectToActive(1, AGENT_ID), '10')).to.be.equal('5');
+            expect(await this.staking.inactiveSharesToStake(subjectToInactive(1, AGENT_ID), '10')).to.be.equal('10');
+        });
+    });
+    describe('FortaStaking', async function () {
```

Comment:
Is this related to this PR?

Reply:
Not really but the tests were failing and I had to merge dev to `chore/remove_router` to reproduce the problem. That test came along

Reply:
Oh so this test is already in `dev`? Weird that it shows up in the diff.